### DISCUSSION TITLE
how to fix this problem?

### DISCUSSION THREAD

**Discussion by AsunaAnna** [TrinityCore/TrinityCore#28062] [Q&A]
![image](https://user-images.githubusercontent.com/58216205/175997725-a1e7f284-0adb-4ba4-954c-3110f32121e8.png)

My MySQL database has been started！
![image](https://user-images.githubusercontent.com/58216205/175998168-21f7bc9d-d553-408f-ac2d-8937b3862cbc.png)

**Comment by Aokromes** [comment]
your mysql instalation is broken.

**Reply by AsunaAnna** [reply]
But I can access other databases in mysql through navcat.
